src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: { id: string, title: string, content: string, }) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title, content }) => {
+            return (
+              <li
+                className={classNames(
+                  { 'is-active': id === currTab.id },
+                )}
+                key={id}
+                data-cy="Tab"
+              >
+
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (selectedTabId !== id) {
+                      onTabSelected({ id, title, content });
+                    }
```

Comment:
Move this into separated function, and you actually don't need to check this condition inside handleTab if you do it here